### ISSUE TITLE
About modification of source code

### ISSUE THREAD

**Issue by YSK813** [TheShadowEevee/BDSX-Discord-Chatter-Plugin#16]
I don't have a deep understanding of the ISC license, I would like to ask a question.

My friend is using this source code and wants to change it so that only the commands of other add-ons are not sent.
Can I change it and give it to that friend?

Sorry for the strange English ...

**Comment by TheShadowEevee** [comment]
I see you marked this as completed but I will respond anyway just in case.

The ISC license is used purely to match the licence on BDSX. 

The license states essentially as follows:
- You may use, share, and modify this software without restriction for no cost
- Changes/Derivatives of this code _must_ include a copy of the ISC license, as well as the copyright statement in the LICENSE file in this repo.

So TL;DR there is no issue with making and sharing your change with whoever you want as long as you keep the existing LICENSE file intact, only adding a new copyright line to it if you so choose.

**Comment by YSK813** [comment]
Thank you for replying to my answer!

Yesterday I forked this repository. Next, I wrote "Original Repository (Write this repository URL)" in the README.md file.


I know the problem doesn't ask about the license.
But you answer my question.
I am very grateful for it.



Use this source code to add any system.



**Thank you!!**
